draft-ietf-quic-multipath.md

```diff
@@ -578,39 +550,39 @@ Path Identifiers.
 ### Effect of MP_RETIRE_CONNECTION_ID Frame {#retire-cid-close}
 
 Receiving a MP_RETIRE_CONNECTION_ID frame causes an endpoint to discard
-the resources associated with that Connection ID. Note that retirement of
-Connection IDs will not effect the use of Path Identifier for the specific path.
-The list of received packets used to send acknowledgements is also remain
+the resources associated with that connection ID. Note that retirement of
+connection IDs will not effect the use of Path Identifier for the specific path.
+The list of received packets used to send acknowledgements also remains
 uneffected as the packet number space is associated with a path.
 
-The peer, that sent RETIRE_CONNECTION_ID frame, can keep sending data using
+The peer, that sends the MP_RETIRE_CONNECTION_ID frame, can keep sending data using
 the same IP addresses and UDP ports previously associated with
-that Connection ID, but MUST use a different connection ID when doing so.
+that connection ID, but MUST use a different connection ID when doing so.
```

Comment:
Same here, this is just RFC 9000. I don't think we need to call out anything specifically here.

Reply:
True, we should at least not use normative language here as this is already normatively specified in RFC9000.
```suggestion
that connection ID, but has to use a different connection ID when doing so.
```